utils/core-contracts/contracts/token/ERC20.sol

```diff
@@ -175,6 +175,12 @@ contract ERC20 is IERC20 {
         }
     }
 
+    function _checkZeroAddress(address target) private pure {
```

Comment:
maybe reuse this by calling this function in `_checkZeroAddressOrAmount`.  nonblocking